.gitignore

```diff
@@ -5,6 +5,10 @@
 # pycharm nonsense
 *.idea
 
+# IDE settings
+.vscode/
+.idea/
+
```

Comment:
started doing a bit of refactoring as there were diplicates between the different .gitignores